.github/workflows/buildMain.yml

```diff
@@ -60,9 +60,9 @@ jobs:
         run: sh buildRun.sh -p ${{ matrix.tutorial }} ${{ github.event.inputs.logLevel }}
       
       - name: Repository dispatch
-        uses: peter-evans/repository-dispatch@v1
+        uses: guilouro/multiple-repositories-dispatch@v1
         with:
           token: ${{ secrets.REPO_ACCESS_TOKEN }}
-          repository: devonfw-tutorials/katacoda-scenarios
+          repositories: devonfw-tutorials/wiki-tutorials|devonfw-tutorials/katacoda-scenarios
```

Comment:
repositories: |
      user/repo1
      user/repo2

I am not sure, but I think this is the right syntax

Reply:
[see](https://github.com/guilouro/multiple-repositories-dispatch)